static/js/views/settings.js

```diff
@@ -243,6 +243,8 @@ const SettingsScreen = {
       this.elements.network.client.ethernet.gatewayLabel,
       this.elements.network.client.ethernet.gateway,
     ];
+    // we'll use these in /settings/network/ethernet screen if 'static' IP is configured
+    this.ethernetEls = ethernetEls;
```

Comment:
I haven't edited this part of the code before, but I note that the Settings class maintains a hierarchical map of all of the HTML elements based on their section within the Settings UI. This line creates a set of duplicate references outside of that `this.elements` map.

If this subset of elements needs sharing between methods, would it instead make more sense to create an additional layer of nesting within the `this.elements` data structure (e.g. `this.network.client.ethernet.static`), and iterate through the subsection of the map rather than create a new duplicate array?